grr/server/grr_response_server/gui/ui/components/client_overview/client_overview_test.ts

```diff
@@ -0,0 +1,231 @@
+import {Location} from '@angular/common';
+import {async, TestBed} from '@angular/core/testing';
+import {MatChip, MatChipList} from '@angular/material/chips';
+import {MatDrawer} from '@angular/material/sidenav';
+import {By} from '@angular/platform-browser';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {ActivatedRoute} from '@angular/router';
+import {ApiModule} from '@app/lib/api/module';
+import {Client} from '@app/lib/models/client';
+import {newClient} from '@app/lib/models/model_test_util';
+import {ClientPageFacade} from '@app/store/client_page_facade';
+import {Subject} from 'rxjs';
+
+import {ClientOverview} from './client_overview';
+import {ClientOverviewModule} from './module';
+
+describe('Client Overview', () => {
+  let facade: ClientPageFacade;
+  let location: Location;
+
+  beforeEach(async(() => {
+    TestBed
+        .configureTestingModule({
+          imports: [
+            ApiModule,
+            NoopAnimationsModule,
+            ClientOverviewModule,
+          ],
+          providers: [
+            {
+              provide: ActivatedRoute,
+              useValue: {},
+            },
+          ]
+        })
+        .compileComponents();
+
+    facade = TestBed.inject(ClientPageFacade);
+    location = TestBed.get(Location);
+  }));
+
+  it('displays client details on client change', () => {
+    const subject = new Subject<Client>();
+    Object.defineProperty(facade, 'selectedClient$', {get: () => subject});
+
+    const fixture = TestBed.createComponent(ClientOverview);
+    fixture.detectChanges();  // Ensure ngOnInit hook completes.
+
+    subject.next(newClient({
+      clientId: 'C.1234',
+      knowledgeBase: {
+        fqdn: 'foo.unknown',
+      },
+      lastSeenAt: new Date(1571789996678),
```

Comment:
Is there a point to adding this in the client information if it is not being used in the test assertions?

Reply:
No, I'll delete the unneeded line.